modules/demo/src/main/scala/lucuma/ui/demo/Demo.scala

```diff
@@ -10,12 +10,18 @@ import cats.effect._
 import crystal._
 import crystal.react._
 import crystal.react.implicits._
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.auto._
+import eu.timepit.refined.numeric.Interval
 import japgolly.scalajs.react.Reusability._
 import japgolly.scalajs.react._
 import japgolly.scalajs.react.extra.ReusabilityOverlay
 import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.core.math._
 import lucuma.ui.forms._
+import lucuma.ui.reusability._
 import monocle.macros.Lenses
+import _root_.react.semanticui.collections.form.Form
```

Comment:
Do we need `_root_` here?

Reply:
I just took that from Raúl's PR. 😆 I see he addressed it there, so I will fix it the same way.